tests/unit/aiplatform/test_initializer.py

```diff
@@ -90,6 +91,10 @@ def test_init_location_with_invalid_location_raises(self):
         with pytest.raises(ValueError):
             initializer.global_config.init(location=_TEST_INVALID_LOCATION)
 
+    def test_init_network_sets_network(self):
```

Comment:
This only tests whether the `_network` attribute is correctly set in the config.  Can we add a test where that attribute is used somewhere downstream too?